src/features/views/components/ViewDataTable/index.tsx

```diff
@@ -93,6 +96,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const getFilterOperators = (col: ZetkinViewColumn) => {
+  if (COLUMN_TYPE.LOCAL_BOOL === col.type) {
+    return getGridBooleanOperators();
+  } else {
+    return getGridStringOperators().filter((op) => op.value !== 'isAnyOf');
```

Comment:
This means all columns except `local_bool` get handled as strings, regardless of how they've been marked up.